server/controllers/medical/reports/patient.receipt.handlebars

```diff
@@ -2,19 +2,34 @@
 
 <body class="container">
   <div class="row">
-    <div class="col-xs-8">
-      <h2>
-        <strong>{{patient.display_name}}</strong>
-      </h2>
+
+    <!-- left side is patient information -->
+    <div class="col-xs-7">
+      <h2><strong>{{patient.display_name}}</strong></h2>
+
+      {{translate "FORM.LABELS.DOB"}} <i class="fa fa-{{patient.symbol}}"></i> {{date patient.dob }} <br />
+      {{#if patient.email}}
+        <u>{{translate "FORM.LABELS.EMAIL"}}</u> {{patient.email}}<br />
+      {{/if}}
+
+      {{#if patient.telephone}}
+        <u>{{translate "FORM.LABELS.TELEPHONE"}}</u> {{patient.telephone}}<br />
+      {{/if}}
+
+      <br />
+      <u>{{translate "FORM.LABELS.ADDRESS"}}</u> <br />
+      <address>
+        {{village.name}}, {{village.sector_name}}, {{village.province_name}} <br />
+        {{village.country_name}}
+      </address>
     </div>
-    <div class="col-xs-4" style="text-align:right;">
-      <h2>
-        <strong>{{patient.reference}}</strong>
-      </h2>
+
+    <!-- right side is enterprise information -->
+    <div class="col-xs-5" style="text-align:right;">
+      <h2><strong>{{patient.reference}}</strong></h2>
+      {{patient.enterprise_name}} <br/>
+      <u>{{translate "FORM.LABELS.DEBTOR_GROUP"}}</u> {{ patient.debtor_group_name }}<br />
+      {{translate "TABLE.COLUMNS.REGISTERED_ON"}} {{date patient.registration_date }}<br />
```

Comment:
It's not blocking your PR, what do you think if the debtor group information was on the patient details side instead of the enterprise details side ?